misc/userscripts/readability-js

```diff
@@ -108,6 +117,9 @@ const HEADER = `
     yaDhjMS4xMSAwIDItMC44OTUgMi0ycy0wLjg5NS0yLTItMnoiIGZpbGw9IiNmZmYiLz4KPC9nPgo8L3N2Zz4K"/>
 </head>
 <body class="qute-readability">
+    <h1 class="reader-title">%s</h1>
+    <div>From <a class="reader-title" href=%s>%s</a></div>
```

Comment:
```suggestion
    <div>From <a class="reader-title" href="%s">%s</a></div>
```

I think?

Reply:
Right, nice catch. Let me think about the best solution (e.g. a template engine).

Reply:
I think the double quotes and the templating thing are two different issues. The double quotes are required no matter what for this to be valid HTML.